database/database.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 type Database interface {
-	GetPosts() ([]common.Post, error)
+	GetPosts(int, int) ([]common.Post, error)
```

Comment:
You changed the interface, don't forget to change the tests too! See the `cmd/urchin/index_test.go`